content/geoip/docs/web-services/_schemas/ResponseRegisteredCountry.mdx

```diff
@@ -9,12 +9,18 @@ import responseJson from '../_examples/geoip';
   A JSON object containing details about the country in which the ISP has
   registered the IP address.
 
+  [Learn about registered countries on our Knowledge
```

Comment:
Just noticed that at line 12, instead of learn more, it's missing more. :)

Reply:
yes, in this case i wanted "learn" instead of "learn more," because i don't think it's very clear what a registered or represented country is at all from the API schema